src/bootstrap/test.rs

```diff
@@ -424,6 +424,49 @@ fn path_for_cargo(builder: &Builder, compiler: Compiler) -> OsString {
     env::join_paths(iter::once(path).chain(env::split_paths(&old_path))).expect("")
 }
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub struct RustdocTheme {
+    pub compiler: Compiler,
+    pub host: Interned<String>,
```

Comment:
Why is this not `compiler.host`? It should be afaict... or it should be named differently, or have some documentation. Right now this seems likely to be brittle in some configurations.